### PR TITLE
today exercises and remove method from list3

### PR DESCRIPTION
**Pull request by 11TK11**

### CODE REVIEW

File: src/kevinsanchez/kata_bank/Convert.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+                    string += "6"
+                if matrix[j][i] is 1 and matrix[j+1][i] is 1 and matrix[j+2][i] is 1:
+                    string += "7"
+                if matrix[j][i] is 1 and matrix[j+1][i] is 4 and matrix[j+2][i] is 4:
+                    string += "8"
+                if matrix[j][i] is 1 and matrix[j+1][i] is 4 and matrix[j+2][i] is 2:
+                    string += "9"
+                i += 1
+            list.append(string)
+            j += 3
```
Comment (sperBruno) [inline src/kevinsanchez/kata_bank/Convert.py]: what does j mean?

File: src/kevinsanchez/kata_bank/Convert.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        return list
+
+    """
+    read and set the matrix.
+    :return a list with the number from the file.
+    """
+    def read(self):
+        f = open('../../../source_data.txt', 'r')
```
Comment (sperBruno) [inline src/kevinsanchez/kata_bank/Convert.py]: file path should be a parameter

File: src/kevinsanchez/task3_11_06_2018/ReverseWord.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    :param phrase to reverse.
+    :return phrase with resersed words.
+    """
+    def reverse_alternate(self, phrase):
+        words = phrase.split(" ")
+        new_phrase = ""
+        size = len(words)
+        i = 0
+        while size > i:
+            if i%2 is not 0:
```
Comment (sperBruno) [inline src/kevinsanchez/task3_11_06_2018/ReverseWord.py:17]: format your code

File: src/kevinsanchez/task3_11_06_2018/RomanNumerals.py [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+        return string
+
+    """
+    helps to convert cases like IX, IV, etc.
+    :param f_number first number in special cases.
+    :param s_number second number in special cases.
+    :return the final number of the cases. 
+    """
+    def helper(self, f_number, s_number):
+        if f_number < s_number:
```
Comment (sperBruno) [inline src/kevinsanchez/task3_11_06_2018/RomanNumerals.py]: use this format statement if condition else statement

File: src/kevinsanchez/task3_11_06_2018/RomanNumerals.py [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+    """
+    convert from roman to decimal.
+    :param string roman number.
+    :return decimal number.
+    """
+    def from_roman(self, string):
+        size = len(string)
+        number = 0
+        while size > 1:
+            f = self.mapp.get(string[size-2])
```
Comment (sperBruno) [inline src/kevinsanchez/task3_11_06_2018/RomanNumerals.py]: what does f and s mean?

**Review by sperBruno — COMMENTED** [review]
Please also remove .idea directory

**Review by ghost — APPROVED** [review]

**Pull request merged**